config.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def getenv_string(environment_variable: str) -> str:
+    s = os.getenv(environment_variable)
+
+    if not s:
+        raise Exception(
+            f"Required environment variable {environment_variable!r} is not set. Please read ./ENVIRONMENT.md"
+        )
+
+    return s
```

Comment:
```suggestion
    if s := os.getenv(environment_variable):
        return s
    msg = f"Required environment variable {environment_variable!r} is not set. Please read ./ENVIRONMENT.md"
    raise OSError(msg)
```
Suggesting this for a few reasons:
1. We want to use named expressions to simplify assignment and conditionals
2. Swapping the if-else branches
3. Assigning exception to a variable. Python includes the `raise` in the default traceback. By using an f-string, the error message will be duplicated in the traceback, which can make the traceback less readable.
4. Specifying OSError

Reply:
All sounds good to me, I've never thought about number 3 before, very good point!